### DISCUSSION TITLE
Is the `.rpm` installer tested on (Open)SUSE?

### DISCUSSION THREAD

**Discussion by beedell-roke** [openaudible/openaudible#1259] [Q&A] [answered]
Considering many independent distributions use the `.rpm` format, I'd like to know which distributions' packages they are tested against. Some solely bother with Fedora.

**Comment by beedell-roke** [comment]
> https://github.com/openaudible/openaudible/discussions/1259#discussion-5615478

https://github.com/openaudible/openaudible/discussions/1274#discussion-5688447

**Comment by openaudible** [comment]
We don’t test all the Linux system. I’d recommend using the installer. I did a little google and didn’t see any issues with the rpm but the .sh installer is probably the safest way to install. We use install4j as our installer and it usually works just fine.

**Reply by beedell-roke** [reply]
> https://github.com/openaudible/openaudible/discussions/1259#discussioncomment-8390940

@openaudible, have you seen the advice contained at https://forums.opensuse.org/t/proton-bridge-installation-error/172223/5?

**Comment by openaudible** [comment, marked as answer]
Fixed in latest beta: https://openaudible.org/beta and will be fixed in next release.
